tests/pass/coroutine.rs

```diff
@@ -183,18 +183,18 @@ fn basic() {
 
 fn smoke_resume_arg() {
     fn drain<G: Coroutine<R, Yield = Y> + Unpin, R, Y>(
-        gen: &mut G,
+        gen_: &mut G,
```

Comment:
Why is this change needed? You didn't change the edition used for tests. Is it due to the rustfmt change?

Reply:
Yeah rustfmt needs the files to parse, and if there is a keyword (which it is on 2024), it doesn't like that.